src/schema/spectraTokens.schema.json

```
@@ -0,0 +1,120 @@
+{
+  "type": "array",
+  "uniqueItems": true,
+  "items": {
+    "type": "object",
+    "additionalProperties": false,
+    "required": [
+      "chainId",
+      "address",
+      "name",
+      "symbol",
+      "decimals",
+      "logoURI"
+    ],
+    "properties": {
+      "chainId": {
+        "type": "integer"
+      },
+      "address": {
+        "type": "string",
+        "pattern": "^0x[a-fA-F0-9]{40}$"
+      },
+      "name": {
+        "type": "string"
+      },
+      "symbol": {
+        "type": "string"
+      },
+      "decimals": {
+        "type": "integer",
+        "minimum": 1,
+        "maximum": 18
+      },
+      "logoURI": {
+        "type": "string"
+      },
+      "extensions": {
+        "type": "object",
+        "additionalProperties": false,
+        "properties": {
+          "underlying": {
+            "type": "string",
+            "pattern": "^0x[a-fA-F0-9]{40}$"
+          },
```

```suggestion
          }, "externalLink": { "type": "string", "pattern": "https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)" },
```

Can we add this extension: `externalLink` 🙏

---

we have the `externalLink` already (line 48) but I will extend it by the pattern

---

I had to change the regex and provided one doesn't work for our current value